molecule_linear-progress/src/linear-progress.ts

```diff
@@ -52,12 +55,17 @@ export class LinearProgress extends LitElement {
   protected override render(): TemplateResult {
     return html`
       ${this._displayTitle()}
-      <div class="flex" style="height: ${this.height}; width: ${this.width};">
-        ${this._renderParts()}
-      </div>
+      ${this._renderBar()}
```

Comment:
Shouldn't this use mwc-linear-progress atom ? I'm confused

Reply:
Nop, mwc-linear do not fit with our needs. We have to handle it manually. 

Indeed we need to have multiple values on the progress and mwc-linear progress allow us to only manage 2